planning/obstacle_stop_planner/launch/obstacle_stop_planner.launch.xml

```diff
@@ -0,0 +1,33 @@
+<launch>
+  <arg name="common_param_path" />
```

Comment:
Could you please create a sample param file and add the default status?

Reply:
Fixed at aeeafcb.